docs/concepts/storage/volumes.md

```diff
@@ -696,6 +697,24 @@ spec:
 
 For further detail, plese the see the [ScaleIO examples](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/scaleio).
 
+### local
+This volume type is alpha in 1.7.
+
+A `local` volume represents a mounted local storage device such as a disk,
+partition or directory.
+
+Local volumes can only be used as a statically created PersistentVolume source.
```

Comment:
This doesn't read well. How about `Local volumes can only be used as a statically created PersistentVolumes` or `Local volumes can only be used as a source in statically created PersistentVolumes` ?